### PR TITLE
Update Bambi introduction

### PR DESCRIPTION
**Pull request by canyon289**
Added some more comments and changes for sections 10-50. Theyre fairlly minor. Great work. Youre getting really good at this Tomas!

### CODE REVIEW

File: lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb [hunk 1]
```diff
@@ -4,15 +4,7 @@
    "cell_type": "code",
```
Comment (tomicapretto) [inline lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb:4]: <p>Oh, this is good!</p><p> </p><p>I'm adding a slide with a meme to quickly discuss this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Comment (tomicapretto) [inline lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb:4]: <p>They were split up previously and you mentioned to consolidate into a single slide hahaha. Not complaining though. If you can come up with a split that would be great. If not, I think this is already good.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='105'/>

Comment (tomicapretto) [inline lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb:4]: <p>Small change but it makes sense! thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='125'/>

Comment (tomicapretto) [inline lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb:4]: <p>Thanks! :D </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='192'/>

Comment (tomicapretto) [inline lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb:4]: <p>Thanks for these changes. I'm adding it, just tweaked the formula a little.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='194'/>

Comment (tomicapretto) [inline lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb:4]: <p>I agree the previous title was very improvable. But I think the phrase should be more related to the concept being learned, not about its importance (which is very important of course). What about any of the following? </p><ul><li>"Making parameters unique" </li><li>"Why you want parameters to be unique"</li><li>"You'd better have unique parameters and here's why"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='221'/>

Comment (canyon289) [reply lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb:4]: <p>"Making parameters unique" sounds great</p>

Comment (canyon289) [reply lessons/03_Bambi_Introduction/Bambi_Introduction.ipynb:4]: <p>Ah my apologies about that. its good as is</p>

**Comment by tomicapretto** [comment]
Thanks a lot for all the input! I'm closing it now since changes are already incorporated to the notebook :)

**Pull request closed without merging**